### PR TITLE
Staging to Main Typescript Refactoring for Acc Test Suite

### PR DESCRIPTION
**Pull request by sharon-zhu**
Refer to Staging PR for comments.

### CODE REVIEW

File: src/utils/hooksTypes.ts [hunk 1]
```diff
@@ -25,30 +25,30 @@ export interface Callback {
...
-  draggableStatements?: Array<HooksStatements>;
-  testName?: string;
-  index?: number;
-  text?: string;
-  contextFileName?: string;
-  contextFilePath?: string;
-  assertions?: Assertion;
-  callbackfunc?: Callback;
-}
+// export interface Action {
```
Comment (annieshinn) [inline src/utils/hooksTypes.ts:28]: Let's take these comments out in future PRs :D

File: src/utils/endpointTypes.ts [hunk 2]
```diff
@@ -61,50 +61,31 @@ export interface Header {
...
-
-// export interface HooksTestModalProps extends HooksTestMenuProps {
-//   isHooksModalOpen: boolean;
-//   closeHooksModal: () => void;
-// }
-
 export interface EndpointTestStatementsProps extends EndpointTestMenuProps {
   endpointStatements: Array<EndpointStatements>;
 }
+
```
Comment (annieshinn) [inline src/utils/endpointTypes.ts:70]: Let's remove this in a future PR!

**Review by annieshinn — APPROVED** [review]
Looks good! Approved.

**Review by taoantaoan — APPROVED** [review]
Nice work! It was a beast.

**Review by astaiglesia — APPROVED** [review]
Looks great team!
No comments to leave and merge is approved.

**Pull request merged**